content/docs/user-guide/how-to/untrack-data.md

```diff
@@ -0,0 +1,45 @@
+# How to Un-track Data
```

Comment:
you should use git mv or something like that to move the file? otherwise there is no way to review what has changed in the file. It looks like a new one

Reply:
The .dvc file? It will be deleted by `dvc remove`. So I guess the output of `git status` would be different if the .dvc file was previously committed, but probably no need to focus much on Git here?